.github/container/build-jax.sh

```diff
@@ -316,6 +316,9 @@ pip --disable-pip-version-check install -e ${BUILD_PATH_JAXLIB}/jaxlib -e ${BUIL
 #  jaxlib                  0.4.32.dev20240808           /opt/jaxlibs/jaxlib
 pip list | grep jax
 
+# Ensure directories are readable by all for non-root users
+chmod 755 $BUILD_PATH_JAXLIB/*
```

Comment:
In another thread it was suggested `umask 0022 ` might be a "better" mechanism for setting permissions on the jax directories as `644` for files and `755` for folders. This should resolve the permissions.

Reply:
I think, it's reasonable comment. @terrykong WDYT?

Reply:
Will resolve this for now since I don't have BW to test and chmod should work as a stopgap.